client/lib/cart-values/index.js

```diff
@@ -190,6 +193,13 @@ function isBelgiumBancontactEnabled( cart ) {
 	);
 }
 
+function isPolandP24Enabled( cart ) {
+	return (
+		config.isEnabled( 'upgrades/poland-p24' ) &&
```

Comment:
Does it make sense to have calypso feature flag for this when we can control if it is enabled server side?

Reply:
I think you're right. I'll remove the existing ones in a separate pr, and remove it here too.